tooling/template-base/components/LearnerAccess/LoadWishlistedContent.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useCallback } from 'react';
+import { NetworkStatus } from '@apollo/client';
+import {
+  useUserWaitlistQuery,
+  LoadingDots,
+  useUnenrollFromWaitlistMutation,
+  hydrateContent,
+  GlobalTypes
+} from '@thoughtindustries/content';
+import useLearnerAccess from './Context/use-context';
+import { useTranslation } from 'react-i18next';
+import { t } from 'i18next';
+
+const LoadWaitlist = (): JSX.Element => {
+  const { i18n } = useTranslation();
+  const {
+    data,
+    loading,
+    error,
+    refetch: refetchWaitlist,
+    networkStatus
+  } = useUserWaitlistQuery({
+    variables: {},
+    fetchPolicy: 'network-only',
+    ssr: false,
+    notifyOnNetworkStatusChange: true
+  });
+  const [unenrollFromWaitlistMutation] = useUnenrollFromWaitlistMutation();
+  const { refetchContentGroups, resetActiveTab } = useLearnerAccess();
+  const handleUnenroll = useCallback(
+    async id => {
+      await unenrollFromWaitlistMutation({ variables: { id } });
+      const { data: refetchData } = await refetchWaitlist();
+      if (refetchData && !refetchData.UserWaitlist?.length) {
+        resetActiveTab();
+      }
+      await refetchContentGroups();
+    },
+    [refetchContentGroups, refetchWaitlist, resetActiveTab]
+  );
+  const isRefetching = networkStatus === NetworkStatus.refetch;
+  console.log('data from child', data);
+  if (loading || isRefetching) return <LoadingDots />;
+  if (error) return <>{error.message}</>;
+  if (!data || !data.UserWaitlist) return <></>;
+  return (
+    <div className="py-5">
+      {data.UserWaitlist.map(item => {
+        const { id, title, contentTypeLabel } = hydrateContent(i18n, item as GlobalTypes.Content);
```

Comment:
If you only need `id`, `title` and `contentTypeLabel`, those should already exist on the item and don't require hydration.